plugins/kubernetes/src/api/types.ts

```diff
@@ -26,6 +26,10 @@ export const kubernetesApiRef = createApiRef<KubernetesApi>({
   id: 'plugin.kubernetes.service',
 });
 
+export const kubernetesProxyApiRef = createApiRef<KubernetesProxyApi>({
+  id: 'plugin.kubernetesproxy.service',
```

Comment:
```suggestion
  id: 'plugin.kubernetes.proxy-service',
```

Or something like that right? The plugin is still `kubernetes`